### DISCUSSION TITLE
Inform of repo breakage

### DISCUSSION THREAD

**Discussion by yarons** [WeblateOrg/weblate#8276] [Ideas]
I often run into broken repositories with some repository merge issue.

I was thinking: if it's a GitHub/GitLab integration would it be possible to automatically open an issue in such case?
This way the maintainers will address this issue quicker, most of the time the maintainers are unaware of this situation.

**Comment by nijel** [comment]
With gitlab or GitHub integration you will get a conflicting merge request, I guess that the issue is bigger for repositories without the integration...

**Comment by yarons** [comment]
@nijel Well in that case there should be an option to supply an email address for notifications.

**Reply by nijel** [reply]
You can configure notifications in the profile: https://hosted.weblate.org/accounts/profile/#notifications

**Reply by yarons** [reply]
I'm suggesting this because many maintainers are unaware and it sometimes takes several days until they realize something's not right.

It's either they haven't configured the mail or ignored it.

**Reply by nijel** [reply]
The default admin notifications include such alerts. If they are ignoring the e-mail, how would configuring another e-mail solve this issue?

**Reply by yarons** [reply]
The original idea is to create a PR on their repo, the email is the alternative solution when the remote repo is not on GitHub/GitLab/BitBucket etc. so it can be ignored.

**Reply by nijel** [reply]
PR is created when PRs are configured. The e-mail notification is turned on by default.
